src/main/scala/uk/gov/nationalarchives/downloadfiles/Lambda.scala

```diff
@@ -77,10 +76,12 @@ class Lambda {
             val writeDirectory = originalPath.split("/").init.mkString("/")
             new File(s"$prefix/$writeDirectory").mkdirs()
             val writePath = s"$efsRootLocation/$consignmentId/$originalPath"
+            val dirtyBucketName = record.getS3.getBucket.getName
             val s3Response = fileUtils.writeFileFromS3(writePath, fileId, record, s3).map(_ => {
-              fileFormatSendMessage(DownloadOutput(cognitoId, consignmentId, fileId, originalPath).asJson.noSpaces)
-              antivirusSendMessage(DownloadOutput(cognitoId, consignmentId, fileId, originalPath).asJson.noSpaces)
-              checksumSendMessage(DownloadOutput(cognitoId, consignmentId, fileId, originalPath).asJson.noSpaces)
+              val output = DownloadOutput(consignmentId, fileId, originalPath).asJson.noSpaces
+              fileFormatSendMessage(output)
+              antivirusSendMessage(AntivirusDownloadOutput(consignmentId, fileId, originalPath, userId, userId, dirtyBucketName).asJson.noSpaces)
```

Comment:
Have got the param `userId` twice, is that correct? Should one of them be `cognitoId`

Reply:
The cognitoId and userId are the same value in the old and new buckets, i.e. the first part of the s3 path so technically this is right but it's a bit weird so I've added in another variable for cognitoId with the same value and a comment to remind me to remove it later.

Reply:
Makes sense.